components/ws-manager-bridge/src/prebuild-state-mapper.spec.ts

```diff
@@ -111,8 +112,9 @@ class TestPrebuildStateMapper {
         ];
 
         for (const test of table) {
-            const cut = new PrebuildStateMapper();
-            const actual = cut.mapWorkspaceStatusToPrebuild(test.status as WorkspaceStatus.AsObject);
+            const experimentsClient = getExperimentsClientForBackend();
```

Comment:
Have you run these tests with the feature flag being on? :thinking:
Ideally we'd ran it for both cases, but that might not be worth the effort. :+1:

Reply:
They would fail if we ran them with the ff on, because the expectations differ. We could potentially have a new suite for the new behavior.

Reply:
> We could potentially have a new suite for the new behavior.

Hm, maybe that copy&past is worth it, than. Becuase it should really only differ on the STOPPED vs STOPPING fields in the test inputs. :bubbles: